main.go

```diff
@@ -18,47 +18,55 @@ package main
 
 import (
 	"flag"
-	"log"
-
 	"net/http"
-	_ "net/http/pprof"
-	"os"
-	"runtime/pprof"
 
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
+	cloudredhatcomv1alpha1 "cloud.redhat.com/clowder/v2/apis/cloud.redhat.com/v1alpha1"
 	controllers "cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com"
 	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/clowder_config"
-	// +kubebuilder:scaffold:imports
+	//+kubebuilder:scaffold:imports
 )
 
-var setupLog = ctrl.Log.WithName("setup")
+var (
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
+)
 
-func main() {
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
+	utilruntime.Must(cloudredhatcomv1alpha1.AddToScheme(scheme))
+	//+kubebuilder:scaffold:scheme
+}
+
+func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
-	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+	var probeAddr string
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.Parse()
-
-	if clowder_config.LoadedConfig.DebugOptions.Pprof.CpuFile != "" {
-		f, err := os.Create(clowder_config.LoadedConfig.DebugOptions.Pprof.CpuFile)
-		if err != nil {
-			log.Fatal(err)
-		}
-		pprof.StartCPUProfile(f)
-		defer pprof.StopCPUProfile()
+	opts := zap.Options{
+		Development: true,
```

Comment:
did we flip devmode from false to true in this change?

Reply:
good catch